wp-content/themes/haruoshi_origin/src/scss/style.scss

```
@@ -1,3 +1,30 @@
+:root {
+    --header-height: 10%;
+    --footer-height: 3%;
+  }
+
 body {
-    background-color: coral;
+    height: 2000px;
+    position: relative;
+    background-color: pink;
+}
+
+.container {
+    margin-top: var(--header-height);
```

```suggestion
    margin-top: $header-height;
```
scssの変数の使い方はこんな感じだったはず。違ったら調べてね ❤️